tools/ci/dockerfiles/Dockerfile.kumactl

```diff
@@ -0,0 +1,18 @@
+FROM alpine:3.10
```

Comment:
So why do we have this file? If the motivation is to have kumactl docker image faster, why not do something like this with all images?

Reply:
I've added a new commit to remove old `Dockerfile`s at the repo root.

Now, only 1 `Dockerfile` per component is left.